pkg/volume/vspherevolume/attacher.go

```diff
@@ -184,7 +184,7 @@ func (attacher *vsphereVMDKAttacher) WaitForAttach(spec *volume.Spec, devicePath
 				return path, nil
 			}
 		case <-timer.C:
-			return "", fmt.Errorf("Could not find attached VMDK %q. Timeout waiting for mount paths to be created.", volumeSource.VolumePath)
+			return "", fmt.Errorf("Could not find attached VMDK %q. Timeout waiting for mount paths to be created", volumeSource.VolumePath)
```

Comment:
This should be (wondering why golint doesn't complain):

```suggestion
			return "", fmt.Errorf("could not find attached VMDK %q. Timeout waiting for mount paths to be created", volumeSource.VolumePath)
```

See: https://github.com/golang/go/wiki/CodeReviewComments#error-strings